lottie-compose/src/main/java/com/airbnb/lottie/compose/LottieAnimationState.kt

```diff
@@ -6,25 +6,49 @@ import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
 import androidx.compose.runtime.setValue
 
+/**
+ * Create a [LottieAnimationState] and remember it
+ *
+ * @param autoPlay Whether the animation will auto play
+ * @param repeatCount How many times the animation will be played. Use [Int.MAX_VALUE] for
+ * infinite repetitions.
+ * @param initialProgress The progress the animation starts with
+ * @param enableMergePaths Whether merge paths are enabled
+ */
 @Composable
 fun rememberLottieAnimationState(
     autoPlay: Boolean = true,
     repeatCount: Int = 0,
     initialProgress: Float = 0f,
+    enableMergePaths: Boolean = true
 ): LottieAnimationState {
     // Use rememberSavedInstanceState so you can pause/resume animations
     return remember(repeatCount, autoPlay) {
-        LottieAnimationState(isPlaying = autoPlay, repeatCount, initialProgress)
+        LottieAnimationState(
+            isPlaying = autoPlay,
+            repeatCount = repeatCount,
+            initialProgress = initialProgress,
+            enableMergePaths = enableMergePaths
+        )
     }
 }
 
 /**
+ * State of the [LottieAnimation] composable
+ *
+ * @param autoPlay Whether the animation will auto play
+ * @param repeatCount How many times the animation will be played. Use [Int.MAX_VALUE] for
+ * infinite repetitions.
+ * @param initialProgress The progress the animation starts with
+ * @param enableMergePaths Whether merge paths are enabled
```

Comment:
You can make these docs `Initial value for [propName]` so they are directly linked and the full docs can be there.